DEVELOPMENT.md

```diff
@@ -117,21 +117,44 @@ Your user must be a cluster admin to perform the setup needed for Knative.
 
 The value you use depends on
 [your cluster setup](https://www.knative.dev/docs/install/): when using
-Minikube, the user is your local user; when using GKE, the user is your GCP
-user.
+Minikube or the Kubernetes under local Docker desktop, the user is your local user;
+when using GKE, the user is your GCP user.
 
 ```shell
 # For GCP
 kubectl create clusterrolebinding cluster-admin-binding \
   --clusterrole=cluster-admin \
   --user=$(gcloud config get-value core/account)
 
-# For minikube
+# For minikube or Kubernetes under docker-desktop
 kubectl create clusterrolebinding cluster-admin-binding \
   --clusterrole=cluster-admin \
   --user=$USER
 ```
 
+### Resource allocation for Kubernetes
```

Comment:
Shouldn't this be covered by the cluster provisioning logic, which I believe lives in `knative/docs`?

Reply:
@mattmoor, where is the cluster provisioning logic? Do you have a link? If we need to change knative/docs, where should we change? If knative/docs has described how to allocate the resources, we need to at least add the reference link here, to avoid future confusions.

Reply:
@mattmoor, where is the cluster provisioning logic? Do you have a link? If we need to change knative/docs, where should we change? If knative/docs has described how to allocate the resources, we need to at least add the reference link here, to avoid future confusions.

Reply:
I believe Matt is referring to [docs/install/Knative-with-Docker-for-Mac](https://github.com/knative/docs/blob/master/docs/install/Knative-with-Docker-for-Mac.md) and [docs/install/Knative-with-Minikube](https://github.com/knative/docs/blob/master/docs/install/Knative-with-Minikube.md) guides which are linked from "Set up a kubernetes cluster".

It looks like both of them suggest 4 vCPU / 8 GB memory which sounds like it is no longer sufficient, and it seems like they are likely not obvious enough to find during setup.

Reply:
Right. We have to increase at least for now the number CPUs. I can change the doc there according. Meanwhile, add a reference link here, so folks won't miss it.

Reply:
Updated.